Source/AFError.swift

```diff
@@ -452,7 +538,7 @@ extension AFError: LocalizedError {
             return reason.localizedDescription
         case .requestRetryFailed(let retryError, let originalError):
             return """
-                   Request retry failed with retry error: \(retryError.localizedDescription), \
+                   Request retry failed with retry error: \(retryError.localizedDescription),\
```

Comment:
Please put the space back, it's necessary for the single continuous line output.

Reply:
Done.